### ISSUE TITLE
Provide GeoPointValue in Datastore v1beta3

### ISSUE THREAD

**Issue by ajkannan** [GoogleCloudPlatform/gcloud-java#248]
The geo point value is a new value type supported by Datastore v1beta3.

**Comment by kvanvranken** [comment]
+1 :D

**Comment by ajkannan** [comment]
This is addressed in #270, and will be merged into the master branch once the new Datastore version is ready.
